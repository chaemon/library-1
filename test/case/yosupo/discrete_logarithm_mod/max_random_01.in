100
136987924 40280999 920999146
805152546 234164454 970079859
410134225 668461222 929099933
394300887 86086437 837472497
221329455 57893312 834110599
154204049 738660729 847510959
551666498 540440394 991386817
512342480 452470821 965573287
340240729 144902419 932751791
289571086 423893886 886035524
464748969 105872436 922798503
242780928 505538017 923582647
745991283 983136444 999537013
842202106 646596193 865304379
630412665 134706415 940145001
917006962 498351627 970189559
784928508 545206283 878228861
127534371 580361498 807852449
833497706 569752320 865765963
687931378 25556222 968745739
903079778 244718550 932861208
923575834 20801451 959787665
144327567 355924229 991637662
767675763 39770237 982611662
35094958 619519152 953146245
334280933 412368057 853755702
696252934 727024412 940772804
739609644 879769415 961213819
429336930 578822041 884017875
192014409 783587083 822940665
641105035 776026757 809209758
14794716 647713890 915129181
720571222 16609338 889586530
796633468 641040997 829190516
956678858 9273458 963448975
235611799 713122606 868837259
230441676 782661943 947463366
618863044 263420409 919932537
341598454 346283651 827284241
693782514 787563199 901508939
558988762 325699987 919916238
484659329 514686813 939953398
168335178 452720553 809813066
152089336 237462880 811844689
768682370 643437693 853724989
50514865 225072892 843682975
432384534 267319887 992229303
470850912 634063725 939610404
170224154 233237753 830265497
363661868 30383316 842086119
450267291 509435980 969764888
871736242 398793145 901046261
108164902 535373865 905341558
796833592 454223174 992210044
621479206 158107041 935552687
767759721 136585361 804116358
61417522 286340474 901585027
684185411 747165460 857584824
201499817 652532397 940519367
403721689 163056092 860966053
716321675 768003151 924723830
568646343 716350006 853168408
528630621 313480942 982648655
359276034 829011041 918564698
725160671 666415978 965016458
659739773 31924187 914167505
560410488 130521101 855285186
758692855 367621879 897583846
700477795 673784691 837543666
71823458 547390766 847104797
269296694 866684149 869993466
918907490 292588975 952701522
237458599 110596904 812264755
281530082 3734782 949516013
802111064 342905930 832577044
489807159 227640803 897675158
715231872 928498199 952130560
849120860 925374177 950318222
398434867 729447157 944892127
452366143 469546842 884979530
387659516 713639740 802322781
338125822 835065313 851990136
510544637 841323884 995748304
263447616 97154821 887057106
626682906 850977848 957189035
547354125 291373786 859841009
470490191 110958524 933880303
379219571 498680309 999548360
100482924 442510270 893420986
659400969 841728390 852038415
187583662 175274604 955603163
561050789 483834181 805579665
281236448 744939400 929764253
339209101 825332344 869131066
105053325 334942889 940904604
52772808 89618199 964139976
312651094 267282041 862382450
153189966 645498866 808490626
624826185 89137985 836924940
215551309 168847257 810549394
