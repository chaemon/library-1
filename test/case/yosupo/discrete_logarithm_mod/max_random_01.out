-1
-1
-1
130090915
-1
-1
-1
-1
-1
-1
-1
-1
178900930
-1
-1
-1
262816580
-1
-1
-1
-1
131702274
-1
-1
-1
-1
-1
-1
-1
-1
-1
45967617
-1
-1
66146541
-1
-1
-1
-1
76948595
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
104794987
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
26311146
-1
-1
